bcl/src/bc_cmwx1zzabz.c

```diff
@@ -113,6 +128,56 @@ static void _bc_cmwx1zzabz_task(void *param)
 
                 return;
             }
+            case BC_CMWX1ZZABZ_STATE_RECEIVE:
+            {
+                self->_state = BC_CMWX1ZZABZ_STATE_IDLE;
+
+                while (_bc_cmwx1zzabz_read_response(self))
+                {
+                    bool recv_data = (memcmp(self->_response, "+RECV=", 5) == 0);
```

Comment:
Redundant parenthesis + I suggest to explicitly use association: `a == 0 ? true : false`